<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApplication</class>
 <widget class="QWidget" name="MainApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainApplication</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="ServerChatTab">
      <attribute name="title">
       <string>Server Chat</string>
      </attribute>
      <widget class="QListView" name="ServerChatView">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>471</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ClientMessageEditBox">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>531</y>
         <width>461</width>
         <height>24</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Enter your message here</string>
       </property>
      </widget>
      <widget class="QLabel" name="StatusLabel">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>140</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="LabelToUser">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>230</y>
         <width>481</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>100</y>
         <width>82</width>
         <height>450</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="ConnectButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>388</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="SendButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="FriendListTab">
      <attribute name="title">
       <string>Friend List</string>
      </attribute>
      <widget class="QListWidget" name="ServerMembersList">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>281</width>
         <height>331</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Server Members</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SendPrivateMessageButton">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>90</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Send Private Message</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
